pipelines/mnist/model.py

```diff
@@ -0,0 +1,207 @@
+#  Copyright 2016 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Is this file any different from the one in the mnist example? I think we should avoid having copies of the same code in multiple places for maintenance reasons

Reply:
Yes, you are right. I copied the model.py and updated that to let read args from command line instead of env variables. And now, that can be removed since the PR has been merged.